src/plugins/simulator/model/simulator-content.ts

```diff
@@ -46,6 +47,11 @@ export const SimulatorContentModel = TileContentModel
       return firstSharedModel as SharedVariablesType;
     },
     get simulationData() {
+      // If no simulation has been specified, use the default simulation from appConfig
+      if (!self.simulation) {
+        const defaultSimulation = getAppConfig(self)?.getSetting("defaultSimulation", "simulator");
+        self.simulation = defaultSimulation as string ?? kBrainwavesKey;
```

Comment:
If you default to the first entry or have a `fallbackDefault` defined in `simulations.ts` then you could eliminate the explicit dependency on brainwaves.